lib/Chat/CommentsManager.php

```diff
@@ -25,10 +25,28 @@
 
 use OC\Comments\Comment;
 use OC\Comments\Manager;
+use OCA\Talk\Room;
 use OCP\Comments\IComment;
+use OCP\Comments\NotFoundException;
 use OCP\DB\QueryBuilder\IQueryBuilder;
 
 class CommentsManager extends Manager {
+	/**
+	 * @param Room $chat
+	 * @param string $messageId
+	 * @return IComment
+	 * @throws NotFoundException
+	 */
+	public function getComment(Room $chat, string $messageId): IComment {
```

Comment:
This class is only there so we can have things before they are merged upstream. Anything that is being added here should be upstreamed. This method is talk related and should therefore be in the ChatManager